src/components/SwatchPaletteWidget/_swatch-palette-widget.scss

```diff
@@ -49,10 +49,6 @@
     padding: 0 $spacing-05;
     background: $ui-01;
     top: 0;
-
-    @include carbon--breakpoint('md') {
-      padding-top: $spacing-09;
-    }
```

Comment:
removing this ends up is hiding the sticky tabs when you scroll down. I think instead of removing it if you just remove the medium breakpoint include and add the spacing for everywhere it will "stick" in the right spot.

Reply:
it looks like my git patch didn't include the remaining positioning changes. I changed the hard coded inline `top: 0px` style on the swatch palette widget (in `color.mdx`) to 3rem, although I'm not sure why we inlined the styles initially and also default it to 0px in `<SwatchPaletteWidget>`

Reply:
This was of the built by contractor components that got moved over, so there are a lot of "I'm  not sure why we did this moments..."